geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -0,0 +1,50 @@
+package org.orbisgis
+
+import groovy.transform.BaseScript
+import org.orbisgis.datamanager.JdbcDataSource
+import org.orbisgis.datamanagerapi.dataset.ITable
+import org.orbisgis.processmanagerapi.IProcess
+
+@BaseScript Geoclimate geoclimate
+
+/**
+ * Process used to compute the sum of all building free facades (roofs are excluded) included in a
+ * Reference Spatial Unit (RSU - such as urban blocks) divided by the RSU area. The calculation is performed
+ * according to a building Table where are stored the "building_contiguity", the building wall height and
+ * the "building_total_facade_length" values as well as a correlation Table between buildings and blocks.
+ *
+ * @return A database table name.
+ * @author Jérémy Bernard
+ */
+static IProcess rsuFreeExternalFacadeDensity() {
+return processFactory.create(
+        "RSU free external facade density",
+        [inputBuildingTableName: String,inputFields:String[],inputCorrelationTableName: String,
+         buContiguityFieldName: String, buTotalFacadeLength: String, outputTableName: String, datasource: JdbcDataSource],
```

Comment:
buTotalFacadeLength - > field name ?
buContiguityFieldName -> inputFieldName ?

Reply:
Both are input field names (we should have calculated the buildingTotalFacadeLength and the buildingContiguity to calculate the rsuFreeExternalFacadeDensity - at least it is the fastest solution). If I only described them as inputFieldName we can not differentiate them.

 Is buTotalFacadeLengthFieldName and buContiguityFieldName OK ?